src/models/Accommodation.js

```diff
@@ -78,6 +78,10 @@ class Accommodation {
     @JsonType(Number)
     vacanciesTotal = 1;
 
+    @JsonProperty()
+    @JsonType(Number)
+    vacanciesTaken = 0;
+
```

Comment:
Please do not add this field. Adding this means plainToClass will try to look for such field in the API model, which is wrong, as the API has no such field (it will, later on, but not now). This is the role of the getter / setter below. Plus, it works fine.